src/main/java/com/napier/sem/App.java

```diff
@@ -24,6 +26,13 @@ public static void main(String[] args) {
         System.out.println(population.districtPopulation());
         System.out.println(population.cityPopulation());
 
+        System.out.println(country.worldCountries());
+        System.out.println(country.continentCountries());
+        System.out.println(country.regionCountries());
+        System.out.println(country.topNPopulatedCountries(5));
```

Comment:
These are magic numbers https://en.wikipedia.org/wiki/Magic_number_(programming), define them as `static final` at the top of the class